### PR TITLE
Chore: SCSS cleanup

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- migrates `_normalize.scss`
- migrates `_explore.scss`
- migrates `_json_explorer.scss`

**Why do we need this feature?**

- clean up old deprecated SCSS 🧹 

**Who is this feature for?**

- grafana devs

**Which issue(s) does this PR fix?**:

For https://github.com/grafana/grafana/issues/64129

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/plugins/datasource/loki/components/LokiCheatSheet.tsx [hunk 1]
```diff
@@ -37,7 +39,10 @@ const LOGQL_EXAMPLES = [
   },
 ];
```
Comment (matyax) [inline public/app/plugins/datasource/loki/components/LokiCheatSheet.tsx:40]: Thanks a lot for this clean up! I wasn't aware we still had these legacy styles.

**Review by matyax — COMMENTED** [review]
Logs changes look good!

File: packages/grafana-prometheus/src/components/PromCheatSheet.tsx [hunk 2]
```diff
@@ -29,23 +31,44 @@ const CHEAT_SHEET_ITEMS = [
...
+          ) : null}
+          {item.label}
+        </div>
+      ))}
+    </div>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  cheatSheetItem: css({
```
Comment (tskarhed) [inline packages/grafana-prometheus/src/components/PromCheatSheet.tsx:60]: I am seeing a lot of cheat sheets. Should these be standardized? Or at least share styles to some extent?

Comment (ashharrison90) [reply packages/grafana-prometheus/src/components/PromCheatSheet.tsx:60]: probably, but i think that's then up to the datasources tbh
there's also a question of where that component would live, cause it shouldn't be `@grafana/ui` 😅

**Review by joshhunt — APPROVED** [review]

**Pull request merged**